content/source/docs/enterprise/install/uninstall.html.md

```diff
@@ -0,0 +1,229 @@
+	f3777b2b116e9734fa8e10514caeb1f51f4a1770896094fbb74904346fa5f18b
+	redis
+	e9c8c5566302e65db0d39c1c7c8ca5d886599f535eec5c655b9e99eac73042e8
+	5a41d77a1085b1fccd450804484f989e9bc5c21351666e7dcfe4d20a56c4fd4b
+	090ed8544eb926967c55d5343bc291d14a95534fb6ed3d7c2cd510af83c11786
+	79e2e42e6a316129c0c9ff0314c774652b1c8d70029385ec55583cd6efbae8a8
+	nomad-workers
+	b0c7e5919e3185bf09a15fd8e466db614a82aaf6987f38ac656a396561499f30
+	cc70bd77c11fedf6d0b656eb474e83893fd9a0d0db6e421686adf2defb4eade1
+	aux
+	c749451acfc48aa4b355248d90aaf667c7ab2bc1818fadfbd9fd91b7c9710b34
+	09a6d3e6ac704ac1e151ae925f0bb25ce24f5a330b9d2ffd69f2a9fa0db1abc9
+	
+	Deleted Images:
+	untagged: hashicorp/build-worker:now
+	deleted: sha256:378b3ecd0a947d834964ab4f690189923c884417d6c9a6fa58989b99330c570f
+	deleted: sha256:c7155461ae7897540d42450b157c5584a3ca53a92c0fca151c1b163fd6458176
+	deleted: sha256:82c7cbdecbe94986e0a3880702679a42badfc925eb81a5f0e1d23be4a1c4375d
+	untagged: 10.1.0.20:9874/hashicorp-ptfe-rabbitmq:9e22de8
+	untagged: registry.replicated.com/terraformenterprise/uagwz2oacr7rk.hashicorp-ptfe-rabbitmq:9e22de8
+	untagged: registry.replicated.com/terraformenterprise/uagwz2oacr7rk.hashicorp-ptfe-rabbitmq@sha256:b70a3d010ff77616b1036d96942aa016973ef58c60faf3030c94ceb84cd2867b
+	deleted: sha256:11de65e463132236bd73df48a8ffa490ff2b525a6fadf8f89e876b7a50c59efc
+	deleted: sha256:e50dc68f15e464a836e20ff11ce3d436c0950ffdf81669987a91b5f67a600222
+	deleted: sha256:931118288655ef2e98fa288528cfe5bc325a092d9a729e0ed663ac95cb4c3643
+	deleted: sha256:b41fbc32924dfe39aa506aacc31029ac0cae428e0afc48ce710abadf6c3c248b
+	deleted: sha256:d2fafd954f9d13009ae22483bc6767f8c13621f0a00fe7516f6a53ca02b872e9
+	deleted: sha256:11d8011c83b4c445be64bab28e30c8776e69cda90a739663c3f57b55ed83a519
+	deleted: sha256:751e8ca997e66f4b7892df7c90f70933380598e9c32394ff787642988272a679
+	deleted: sha256:56cedf40e0080b58fe0bda77607e4321b658f6486de25f6750334e60827d239c
+	deleted: sha256:5987af1ce8b66c0ac9bc8fca0a58b29dcf283e756bafad1b9347a1e6a6f907e3
+	deleted: sha256:58f67ec54ffec23435af7a061e142547586ad71d2ca083845bc3b5a024965143
+	untagged: 10.1.0.20:9874/hashicorp-tfe-telegraf:1.16.3-alpine
+	...
+
+	Total reclaimed space: 6.168GB
+	Done.
+	Removing the Replicated and TFE Docker networks...
+	Error: No such network: replicated_retraced
+	Error: No such network: tfe_services
+	Error: No such network: tfe_terraform_isolation
+	Unable to remove all Docker application networks, or none to be removed.
+	Done.
+	Removing any dangling Docker volumes...
+	"docker volume rm" requires at least 1 argument.
+	See 'docker volume rm --help'.
+	
+	Usage:  docker volume rm [OPTIONS] VOLUME [VOLUME...]
+	
+	Remove one or more volumes
+	Unable to remove dangling Docker volumes, or none to be removed.
+	
+	Uninstall Complete
+	```
+
+	You may see errors such as the ones above, these are normal and just mean the script is attempting to clean up something that's already been removed.
```

Comment:
I like this and I would move it up to be right before the "Terraform has been removed from the system." I worry that it's a bit hidden down here and I want folks to see this information about the errors.